### PR TITLE
Add how-to on GIF recording

### PR DESCRIPTION
**Pull request by jwbuurlage**

### CODE REVIEW

File: how-to-guide.org [hunk 1]
```diff
@@ -162,8 +155,17 @@ git push origin reverse_neutron_flow
...
+For recording animated GIFs of a region of your screen, you can use peek
+<https://github.com/phw/peek>. The easiest way to get it to run on a CWI
+workstation is by using an AppImage of a recent release, which you can get from
+the releases page <https://github.com/phw/peek/releases>.
+
+After downloading, make the file executable (either using your file manager, or
+by calling =chmod +x [peek_release.AppImage]= from a terminal). Now you can run
+Peek, which has a straightforward interface: resize the Peek window and click on
+record, after three seconds it will start recording the region of your screen
+that is visible.
```
Comment (ahendriksen) [inline how-to-guide.org:168]: Can you add a GIF explaining how to use peek? Can Peek peek at itself?

File: how-to-guide.org [hunk 2]
```diff
@@ -162,8 +155,17 @@ git push origin reverse_neutron_flow
 #+END_SRC
 
 Visit the project page on GitHub, and in the tab /Pull requests/ click on /New pull request/. For more information about pull requests, see https://help.github.com/articles/about-pull-requests/.
-
-
+*** Record an animated GIF of your screen
+For recording animated GIFs of a region of your screen, you can use peek
```
Comment (ahendriksen) [inline how-to-guide.org]: A small capitalization note: is peek not capitalized as Peek?

**Comment by ahendriksen** [comment]
@jwbuurlage , frankly this looks great. I think you have done a fantastic job explaining a tool that is bound to come in useful for many. 

I have added some small comments, I hope you find them useful.

**Comment by jwbuurlage** [comment]
Haha, so now I am making capitalization errors..

**Comment by jwbuurlage** [comment]
I don't know if you are serious about the GIF, but this would only render on the HTML version.

**Pull request merged**